src/components/Blinds/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./style.css"
+import { useState } from "react"
+
+import blindsOpen from "./blinds-open.svg"
+import blindsClosed from "./blinds-closed.svg"
+
+export const Blinds = ({ state }) => {
+    const [stateBlinds, setStateBlinds] = useState(state)
+
+    const open = () => {
+        setStateBlinds('open')
+    }
+
+    const closed = () => {
+        setStateBlinds('closed')
+    }
```

Comment:
pro funkci se většinou používá sloveso, jakože dělá to nějaku akci. tj "close" (zavři)